css/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #ffe4e1;
+}
+
+header {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 0;
+  padding: 20px;
+}
+
+.logo {
+  font-family: sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.main-page {
+  display: flex;
+  flex-flow: column nowrap;
+  padding: 20px 1.2rem;
+  margin: 0;
+  background-image: url(../images/bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.main-page-h1 {
+  font-size: 36px;
+  font-weight: 700;
+}
+
+.main-p {
+  font-size: 20px;
+  padding: 10px;
+}
```

Comment:
- [ ] Please check the font families for `<h1> <h3> <p>` tags on the Figma design and add it them on your page as well. Eg.: `<p>` tag on Figma has "Poppips" font family. Here is how you can check properties on Figma:

![chrome-capture (8)](https://user-images.githubusercontent.com/77038610/143384339-2e089e58-5e6c-4cd6-8845-a6c52e50854f.gif)

- Click on elements and check the right menu, there you will see all the specifications 🤗